src/commands/hardis/org/diagnose/unusedusers.ts

```diff
@@ -71,25 +75,79 @@ export default class DiagnoseUnusedUsers extends SfdxCommand {
     this.debugMode = this.flags.debug || false;
     this.outputFile = this.flags.outputfile || null;
     this.lastNdays = this.flags.days;
-    this.debugMode = this.flags.debug || false;
-    this.outputFile = this.flags.outputfile || null;
-
+    this.licenseTypes = this.flags.licensetypes;
     // If manual mode and days not sent as parameter, prompt user
-    if (!isCI && !this.lastNdays) {
+    if (!isCI) {
+      if(!this.lastNdays){
+        const lastNdaysResponse = await prompts({
+          type: "select",
+          name: "days",
+          message: "Please select the period to detect inactive active users.",
+          choices: [
+            { title: `7 days`, value: 7 },
+            { title: `30 days`, value: 30 },
+            { title: `90 days`, value: 90 },
+            { title: `6 months (180 days)`, value: 180 },
+            { title: `1 year (365 days)`, value: 365 },
+            { title: `2 years (730 days)`, value: 730 }
+          ],
+        });
+        this.lastNdays = lastNdaysResponse.days;
+      }
+
+      if(!this.licenseTypes){
+        const licenseTypesResponse = await prompts({
+          type: "select",
+          name: "licensetypes",
+          message: "Please select the period to detect inactive active users.",
+          choices: [
+            { title: `all-crm`, value: 'SFDC,AUL,AUL1,AULL_IGHT' },
+            { title: `all-paying`, value: 'SFDC,AUL,AUL1,AULL_IGHT,PID_Customer_Community,PID_Customer_Community_Login,PID_Partner_Community,PID_Partner_Community_Login' },
+            { title: `all`, value: 'AUL,AUL1,AUL_LIGHT,FDC_ONE,FDC_SUB,Overage_Platform_Portal_User,PID_STRATEGIC_PRM,PID_CHATTER,PID_CONTENT,PID_Customer_Portal_Basic,PID_Customer_Portal_Standard,PID_FDC_FREE,PID_IDEAS,PID_Ideas_Only_Portal,PID_Ideas_Only_Site,PID_KNOWLEDGE,PID_Customer_Community,PID_Customer_Community_Login,PID_Partner_Community,PID_Partner_Community_Login,PID_Overage_High Volume Customer Portal,PID_Overage_Customer_Portal_Basic,PID_Limited_Customer_Portal_Standard,PID_Limited_Customer_Portal_Basic,Platform_Portal_User,POWER_SSP,POWER_PRM,SFDC' }
```

Comment:
Just use value "all" when "all" is selected, and remove the criteria from the query :)

Reply:
But it can be nice to let the whole list in the documentation, indeed :)